helm/messenger/README.md

```diff
@@ -0,0 +1,36 @@
+Messenger Helm chart
+====================
+
+[Changelog](https://github.com/team113/messenger/blob/main/helm/messenger/CHANGELOG.md)
+
+
+
+
+## Installation
+
+Add `@messenger` [Helm] charts repository:
+```shell
+helm repo add messenger https://team113.github.io/messenger/helm
+```
+
+See [values file][values] for details.
+
+
+
+
+## License
+
+Copyright © 2022 IT ENGINEERING MANAGEMENT INC, <https://github.com/team113>
+
+This program is free software: you can redistribute it and/or modify it under the terms of the [GNU Affero General Public License v3.0] as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```

Comment:
@SleepySquash well, this requires to add LICENSE headers to Helm chart sources.

Also, this should be mentioned in `Chart.yaml` metadata.

Reply:
@tyranron 

> Also, this should be mentioned in Chart.yaml metadata.

I wasn't able to find any key suited for license notices in the [`Chart.yaml` file itself](https://helm.sh/docs/topics/charts/#the-chartyaml-file), however [structure documentation](https://helm.sh/docs/topics/charts/#the-chartyaml-file) states that the license should be placed as a separate `LICENSE` file (without the extension), so I did exactly that. Am I missing something or is it OK?

Reply:
@SleepySquash it's OK. Additionally, you may add [`artifacthub.io/license` annotation](https://artifacthub.io/docs/topics/annotations/helm/).